src/__swaps__/screens/Swap/hooks/useSwapInputsController.ts

```diff
@@ -135,33 +133,31 @@ export function useSwapInputsController({
       return addCommasToNumber(inputValues.value.inputAmount, '0');
     }
 
-    const assetBalanceDisplay = internalSelectedInputAsset.value.balance.display ?? '';
-
     if (inputMethod.value === 'outputAmount') {
       return valueBasedDecimalFormatter({
         amount: inputValues.value.inputAmount,
         nativePrice: inputNativePrice.value,
-        assetBalanceDisplay,
         roundingMode: 'up',
         precisionAdjustment: -1,
         isStablecoin: internalSelectedInputAsset.value?.type === 'stablecoin' ?? false,
         stripSeparators: false,
       });
     }
 
-    const balance = internalSelectedInputAsset.value?.maxSwappableAmount || 0;
-    const isStablecoin = internalSelectedInputAsset.value?.type === 'stablecoin' ?? false;
+    if (percentageToSwap.value === 1) {
```

Comment:
when it is a max swap, the formatted amount just needs to format the input value which was already set to be the `maxSwappableAmount`